documentation/code-repos.md

```diff
@@ -54,6 +54,10 @@ Almost all HOCS code is open-source and publicly available on GitHub.
 
 [hocs-github-actions](https://github.com/UKHomeOffice/hocs-github-actions)
 
+[hocs-helm-charts](https://github.com/UKHomeOffice/hocs-helm-charts)
+
+[hocs-helmfile](https://github.com/UKHomeOffice/hocs-helmfile)
```

Comment:
This repository is empty - is there plans for this? I've not heard anything about this repo

Reply:
https://github.com/UKHomeOffice/hocs-helmfile 💯